datacore/__init__.py,sha256=iXkuPjsAb4XH6harjB7fnmnU7Mr4RIfvvah_hJRyUkI,369
datacore/datacore_restlike.py,sha256=58EVJLSEOXLibcPb5GlR-P1jqileYKKQ9PPdIaKCjvM,55147
datacore/exceptions.py,sha256=QezEcA3PqwAL0JPrieAOYXSU6pEP0vcDTez5UHO7zys,2293
datacore/types.py,sha256=qQdX2sxdfhTzPJcOrsBGvBx-_IvH4RXfcsE8YQ_wJ2s,1260
datacore_python_client-1.1.6.dist-info/DESCRIPTION.rst,sha256=OCTuuN6LcWulhHS3d5rfjdsQtW22n7HENFRh6jC6ego,10
datacore_python_client-1.1.6.dist-info/METADATA,sha256=FugCiNgNC2J_MNdXv7NtIPcl0hdegdnllBG5rILzW4I,452
datacore_python_client-1.1.6.dist-info/RECORD,,
datacore_python_client-1.1.6.dist-info/WHEEL,sha256=rNo05PbNqwnXiIHFsYm0m22u4Zm6YJtugFG2THx4w3g,92
datacore_python_client-1.1.6.dist-info/metadata.json,sha256=NZtckgw_ZypgsFle0gmXlUPxrC1mnWYeFI0IIUrJWZ8,654
datacore_python_client-1.1.6.dist-info/top_level.txt,sha256=Rm_HpdZaQHkZIHrz7jvVe5kCOvOYmffPHK1pJ0G9hpI,9
